src/main/kotlin/calculator/Token.kt

```diff
@@ -0,0 +1,22 @@
+package calculator
+
+class Token(input: String) {
+
+    val values: List<String>
+
+    init {
+        require(input.isNotBlank()) { "문자열은 공백일 수 없습니다. input: $input" }
+
+        values = REGEX.find(input)?.let {
+            val delimiter = it.groupValues[1]
+            val tokenStr = it.groupValues[2]
```

Comment:
코틀린에서는 구조 분해 선언(destructuring declaration)을 사용하면 복합적인 값을 분해해서 여러 변수에 나눠 담을 수 있습니다. 아래의 글이 도움 될 거예요.
https://kotlinlang.org/docs/destructuring-declarations.html

```suggestion
            val (delimiter, tokenStr) = it.destructured
```